schema/encoding/jsonschema/jsonschema.go

```diff
@@ -135,6 +160,34 @@ func validatorToJSONSchema(w io.Writer, v schema.FieldValidator) (err error) {
 	return ew.err
 }
 
+func serializeField(ew errWriter, key string, field schema.Field) error {
+	ew.writeFormat("%q: {", key)
+	if field.Description != "" {
+		ew.Comma()
+		ew.writeFormat(`"description": %q`, field.Description)
+	}
+	if field.ReadOnly {
+		ew.Comma()
+		ew.writeFormat(`"readOnly": %t`, field.ReadOnly)
+	}
+	if field.Validator != nil {
+		ew.Comma()
+		ew.err = validatorToJSONSchema(ew, field.Validator)
+	}
+	if field.Default != nil {
+		b, err := json.Marshal(field.Default)
+		if err != nil {
+			return err
+		}
+		ew.Comma()
+		ew.writeString(`"default": `)
+		ew.write(b)
```

Comment:
here you call `ew.write()`, but above you define `func 
(ew errWriter) Write(...)`?